### PR TITLE
Minor fixes for landing page

### PR DESCRIPTION
**Pull request by tyler-romero**
Fix API "Learn More" button, fix `mailto:` tag, add autoupdating copyright.

### CODE REVIEW

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -167,7 +167,7 @@ print(image_query)`;
                   <div className={baseStyles.integrationcompatibilityitem}>
                     <h3 className={baseStyles.title}>API</h3>
                     <p>API to let you access your models in the cloud - no need to run your own models or hardware.</p>
-                    <a href="/python-sdk/docs/getting-started " className={`${baseStyles.cmnButton} ${baseStyles.outline}`}>Learn More</a>
+                    <a href="/python-sdk/docs/api-reference-docs" className={`${baseStyles.cmnButton} ${baseStyles.outline}`}>Learn More</a>
```
Comment (brandon-groundlight) [inline docs/src/pages/index.tsx:170]: Wait, is this right? Is it not `/python-sdk/apireference-docs` (without docs)

**Review by brandon-groundlight — CHANGES_REQUESTED** [review]
I don't think the address for the api-reference docs is right?

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (tyler-romero) [reply docs/src/pages/index.tsx]: Whoops! Fixed

**Review by brandon-groundlight — APPROVED** [review]
LGTM

**Pull request merged**